pkg/common/constant/filter.go

```diff
@@ -40,4 +45,19 @@ const (
 	LogDataBuffer = 5000
 	// console
 	Console = "console"
+
+	DefaultReqTimeout = 10 * time.Second
 )
+
+func ResolveTimeStr2Time(currentV string, defaultV time.Duration) time.Duration {
+	fmt.Printf("timeout parse %s : %d", currentV, defaultV)
```

Comment:
using Logger component print log, determine the log level

Reply:
get